pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -273,7 +273,7 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 			log.Error(err, "cannot get infrastructure name")
 			return reconcile.Result{}, err
 		}
-		extNLBName := infrastructureName + "-test"
+		extNLBName := infrastructureName + "-ext"
```

Comment:
Reason for change? Does this affect the logic changing below?

Reply:
this was found by Will Gordon. Very small change. Does not change logic below.

Reply:
Should this be done in a seperate PR or commit then?